articleview.cc

```diff
@@ -489,87 +511,39 @@ void ArticleView::showDefinition( QString const & word, QStringList const & dict
 
   ui.definition->load( req );
 
-  //QApplication::setOverrideCursor( Qt::WaitCursor );
   ui.definition->setCursor( Qt::WaitCursor );
 }
 
 void ArticleView::showAnticipation()
 {
   ui.definition->setHtml( "" );
   ui.definition->setCursor( Qt::WaitCursor );
-  //QApplication::setOverrideCursor( Qt::WaitCursor );
```

Comment:
Why remove this line and the other identical line above. Unrelated to this PR, I think.